docs/source/run/parameters.rst

```diff
@@ -38,6 +38,88 @@ as command line parameters. Multi-line expressions are allowed if surrounded by
 General parameters
 ------------------
 
+* ``hipace.normalized_units`` (`bool`) optional (default `0`)
+    Using normalized units in the simulation.
```

Comment:
```suggestion
    Whether the simulation uses the normalized unit system commonly used in wakefield acceleration, see e.g. `chapter 2 of this reference <https://iopscience.iop.org/article/10.1088/0741-3335/56/8/084012>`__. Otherwise, the code assumes SI (Système International) unit system.
```